lib/install/validate/validator.go

```diff
@@ -178,6 +200,11 @@ func (v *Validator) Validate(ctx context.Context, input *data.Data) (*config.Vir
 
 }
 
+func (v *Validator) ReValidate(ctx context.Context) {
```

Comment:
You could just write ValidateFirewall if you need additional check here. Otherwise, just public firewall method.
Then you could call the method anytime you want, and then get out issues listed in Validator.